### PR TITLE
Refactor tokens definitions

### PR DESCRIPTION
**Pull request by slavcodev**
Refactoring extracted from #183

### CODE REVIEW

File: openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml [hunk 1]
```diff
@@ -9,12 +9,38 @@ properties:
...
+        readOnly: true
+      last4:
+        description: Payment Card PAN's last 4 digits.
+        type: string
+        readOnly: true
+      brand:
+        readOnly: true
+        allOf:
+          - $ref: ../PaymentCards/PaymentCardBrand.yaml
   billingAddress:
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:44]: Indent?

Comment (slavcodev) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:44]: No. `billingAddress` is in the same level as `paymentInstrument`

Comment (ashkarpetin) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:44]: I'm confused, it's not refactoring it changing token definitions and API

Comment (slavcodev) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:44]: What do you mean?

Comment (slavcodev) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:44]: It's same as it is now

```
paymentInstrument: {pan, cvv}
billingAddress: {firstName}
```

**Review by ashkarpetin — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]

**Pull request merged**